pkg/trace/config/config.go

```diff
@@ -515,7 +505,7 @@ func New() *AgentConfig {
 		Obfuscation:                 &ObfuscationConfig{},
 		MaxResourceLen:              5000,
 
-		GlobalTags: make(map[string]string),
+		GlobalTags: computeGlobalTags(),
```

Comment:
This map gets overwritten in the "serverless trace agent" will that be a problem here? Or is that binary not used in azure function apps?